src/devices/Rangefinder2DWrapper/CMakeLists.txt

```diff
@@ -36,3 +36,70 @@ if(NOT SKIP_Rangefinder2DWrapper)
 
   set_property(TARGET yarp_Rangefinder2DWrapper PROPERTY FOLDER "Plugins/Device")
 endif()
+
+####################################################################################################
+yarp_prepare_plugin(Rangefinder2DWrapperROS
+                    CATEGORY device
+                    TYPE Rangefinder2DWrapperROS
+                    INCLUDE Rangefinder2DWrapperROS.h
+                    DEFAULT ON)
+
+if(NOT SKIP_Rangefinder2DWrapper)
+  yarp_add_plugin(yarp_Rangefinder2DWrapperROS)
+
+  target_sources(yarp_Rangefinder2DWrapperROS PRIVATE Rangefinder2DWrapperROS.cpp
+                                                      Rangefinder2DWrapperROS.h)
+
+  target_link_libraries(yarp_Rangefinder2DWrapperROS PRIVATE YARP::YARP_os
+                                                             YARP::YARP_sig
+                                                             YARP::YARP_dev
+                                                             YARP::YARP_rosmsg)
+  list(APPEND YARP_${YARP_PLUGIN_MASTER}_PRIVATE_DEPS YARP_os
+                                                      YARP_sig
+                                                      YARP_dev
+                                                      YARP_rosmsg)
+
+  yarp_install(TARGETS yarp_Rangefinder2DWrapperROS
+               EXPORT YARP_${YARP_PLUGIN_MASTER}
+               COMPONENT ${YARP_PLUGIN_MASTER}
+               LIBRARY DESTINATION ${YARP_DYNAMIC_PLUGINS_INSTALL_DIR}
+               ARCHIVE DESTINATION ${YARP_STATIC_PLUGINS_INSTALL_DIR}
+               YARP_INI DESTINATION ${YARP_PLUGIN_MANIFESTS_INSTALL_DIR})
+
+  set(YARP_${YARP_PLUGIN_MASTER}_PRIVATE_DEPS ${YARP_${YARP_PLUGIN_MASTER}_PRIVATE_DEPS} PARENT_SCOPE)
+
+  set_property(TARGET yarp_Rangefinder2DWrapperROS PROPERTY FOLDER "Plugins/Device")
+endif()
+
+####################################################################################################
+yarp_prepare_plugin(Rangefinder2DWrapperYARP
+                    CATEGORY device
+                    TYPE Rangefinder2DWrapperYARP
+                    INCLUDE Rangefinder2DWrapperYARP.h
```

Comment:
Since this is a wrapper, I think that this should have `WRAPPER Rangefinder2DWrapperYARP`

Reply:
done